kytos/core/controller.py

```diff
@@ -212,6 +243,7 @@ def stop_controller(self, graceful=True):
         self.started_at = None
         self.unload_napps()
         self.buffers = KytosBuffers()
+        os.remove(LOCK_PATH)
```

Comment:
s/LOCK_PATH/PID_FILE/g.

Also you have to handle exceptions like `FileNotFoundError`